src/gmail.js

```diff
@@ -517,37 +513,37 @@ var Gmail = function(localJQuery) {
     };
 
     api.check.is_priority_inbox = function() {
-        return $(".qh").length > 0;
+        return document.querySelector(".qh") !== null;
     };
 
 
     api.check.is_rapportive_installed = function() {
-        return $("#rapportive-sidebar").length === 1;
+        return document.querySelector("#rapportive-sidebar") !== null;
```

Comment:
Some of these queries check for number of results the query returns.

Would it make more sense to use `document.querySelectorAll()` instead to replace those calls, to avoid having to change the Gmail.js-side "business"-logic in these helper methods?

Reply:
This is the only one query I can confirm which will change its behavior from the logic perspective. 🙂 I can change it to check for only one element like it was before, but I believe that it's either "rapportive installed" and you have the element, or it's not installed and no element is present at all.

Reply:
Fair enough. Lets keep it as is.